deploy/helm_charts/envs/worldbank.yaml

```diff
@@ -49,9 +49,11 @@ serviceGroups:
       memoryLimit: "8G"
 nl:
   enabled: true
-  embeddingsSpec:
-    defaultIndex: "medium_ft"
-    enabledIndexes: ["medium_ft", "sdg_ft", "undata_ft"]
+  env:
+    default_indexes:
+      - medium_ft
+    enabled_indexes:
+      - medium_ft
```

Comment:
intend to remove others?

Reply:
I intended to only keep medium_ft for these instances. For world bank, do you think we should keep sdg and undata?

Reply:
Since worldbank is part of un, maybe keep it there?

Reply:
ok